Content.Server/Administration/Commands/DeleteComponent.cs

```diff
@@ -0,0 +1,47 @@
+﻿#nullable enable
+using Content.Shared.Administration;
+using Robust.Server.Interfaces.Console;
+using Robust.Server.Interfaces.Player;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.IoC;
+
+namespace Content.Server.Administration.Commands
+{
+    [AdminCommand(AdminFlags.Admin)]
+    public class DeleteComponent : IClientCommand
+    {
+        public string Command => "deletecomponent";
+        public string Description => "Deletes all instances of the specified component.";
+        public string Help => $"Usage: {Command} <name>";
+
+        public void Execute(IConsoleShell shell, IPlayerSession? player, string[] args)
+        {
+            switch (args.Length)
+            {
+                case 0:
+                    shell.SendText(player, $"Not enough arguments.\n{Help}");
+                    break;
+                default:
+                    var name = string.Join(" ", args);
+                    var componentFactory = IoCManager.Resolve<IComponentFactory>();
+                    var entityManager = IoCManager.Resolve<IEntityManager>();
+                    var componentType = componentFactory.GetRegistration(name).Type;
```

Comment:
This will throw if you enter a invalid component name, either catch it or add a tryget

Reply:
Done, needs a submodule update.